debezium-core/src/main/java/io/debezium/metrics/Metrics.java

```diff
@@ -86,21 +89,23 @@ public synchronized void register() {
                     break;
                 }
                 catch (InstanceAlreadyExistsException e) {
-                    if (attempt <= REGISTRATION_RETRIES) {
+                    if (attempt < REGISTRATION_RETRIES) {
                         LOGGER.warn(
                                 "Unable to register metrics as an old set with the same name exists, retrying in {} (attempt {} out of {})",
                                 REGISTRATION_RETRY_DELAY, attempt, REGISTRATION_RETRIES);
+                        metronome.pause();
                     }
                     else {
-                        LOGGER.error("Failed to register metrics MBean, metrics will not be available");
+                        LOGGER.error("Failed to register metrics MBean, metrics will not be available, retrying in {} (attempt {} out of {})",
```

Comment:
PLease return back the original message. The retry should not happen after {{else}} branch.

Reply:
@jpechane, welcome. Got it.